Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -151,11 +167,53 @@ private void UpdateRaycast()
             {
                 if (oldFocusedObject != null)
                 {
+                    IInteractable oldInteractable;
+
+                    if (!interactableCache.TryGetValue(oldFocusedObject, out oldInteractable))
+                    {
+                        oldInteractable = oldFocusedObject.GetComponent<IInteractable>();
```

Comment:
Here is a place where we should still have the interactable from the previous frame.